contrib/CMakeLists.txt

```diff
@@ -125,11 +125,17 @@ if (USE_INTERNAL_SSL_LIBRARY)
 endif ()
 
 if (ENABLE_MYSQL AND USE_INTERNAL_MYSQL_LIBRARY)
-    add_subdirectory (mariadb-connector-c-cmake)
-    target_include_directories(mysqlclient BEFORE PRIVATE ${ZLIB_INCLUDE_DIR})
-    if(OPENSSL_INCLUDE_DIR)
-        target_include_directories(mysqlclient BEFORE PRIVATE ${OPENSSL_INCLUDE_DIR})
+    set(CLIENT_PLUGIN_CACHING_SHA2_PASSWORD STATIC)
+    set(CLIENT_PLUGIN_SHA256_PASSWORD STATIC)
+    set(CLIENT_PLUGIN_REMOTE_IO OFF)
+    set(CLIENT_PLUGIN_DIALOG OFF)
+    set(CLIENT_PLUGIN_CLIENT_ED25519 OFF)
+    set(CLIENT_PLUGIN_MYSQL_CLEAR_PASSWORD OFF)
```

Comment:
I cannot find this in https://github.com/ClickHouse-Extras/mariadb-connector-c/blob/18016300b00825a3fcbc6fb2aa37ac3e51416f71/CMakeLists.txt, and there never were such variables.

Am I missing something? Maybe some kind of substitution?

Reply:
Egh, I guess this is https://github.com/ClickHouse-Extras/mariadb-connector-c/blob/18016300b00825a3fcbc6fb2aa37ac3e51416f71/cmake/plugins.cmake here